__tests__/app.test.js

```diff
@@ -97,25 +97,55 @@ describe("Error Handling", () => {
         expect(res.body.msg).toBe("Invalid input");
       });
   });
-    test("status:400, responds with an error message when passed a bad ID", () => {
+  test("status:400, responds with an error message when passed a bad ID", () => {
+    return request(app)
+      .get("/api/articles/abcd/comments")
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe("Invalid input");
+      });
+  });
+
+  test("status:404, responds with an error message when article does not exist", () => {
+    return request(app)
+      .get("/api/articles/1234/comments")
+      .expect(404)
+      .then((res) => {
+        expect(res.body.msg).toBe("No article found by that ID");
+      });
+  });
+
+    test("status:404, responds with an error message when article does not exist", () => {
```

Comment:
I see that this test is different to the one above, but the description line does not reflect that.